### PR TITLE
Release capa v1.6.1

### PR DESCRIPTION
**Pull request by Ana06**
- [X] Ensure all [milestoned issues/PRs](https://github.com/fireeye/capa/milestones) are addressed, or reassign to a new milestone.
- [X] Add the `dont merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/fireeye/capa/pulls) and [capa-rules](https://github.com/fireeye/capa-rules/pulls).
- [X] Ensure the [CI workflow succeeds in master](https://github.com/fireeye/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [X] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples`  offenses are allowed in the nursery).
- [X] Review changes
  - capa https://github.com/fireeye/capa/compare/\<last-release\>...master
  - capa-rules https://github.com/fireeye/capa-rules/compare/\<last-release>\...master
- [X] Update [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections
    - New Features
    - New Rules
    - Bug Fixes
    - Changes
    - Development
    - Raw diffs
- [X] Update [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py)
- [X] Create a PR with the updated [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/fireeye/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/fireeye/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/fireeye/capa/blob/master/CHANGELOG.md).
- [ ] Verify GH actions [upload artifacts](https://github.com/fireeye/capa/releases), [publish to PyPI](https://pypi.org/project/flare-capa) and [create a tag in capa rules](https://github.com/fireeye/capa-rules/tags) upon completion.
- [ ] [Spread the word](https://twitter.com). **Do we do this for minor releases? As we are merging #480 after the release, we could mention both things at the same time**

### CODE REVIEW

File: capa/version.py
Comment (mr-tz) [inline capa/version.py:1]: very important ;)

File: CHANGELOG.md
Comment (mr-tz) [inline CHANGELOG.md:26]: 🥳

**Review by mr-tz — APPROVED** [review]
looks great, let's wait for one more review and then stride quickly towards v2.0!

File: CHANGELOG.md
Comment (williballenthin) [inline CHANGELOG.md:22]:
```suggestion
This release includes several bug fixes, such as a vivisect issue that prevented capa from working on Windows with Python 3. It also adds 17 new rules and a bunch of improvements in the rules and IDA rule generator. We appreciate everyone who opened issues, provided feedback, and contributed code and rules.
```

**Review by williballenthin — APPROVED** [review]
i think these release notes are great, thanks @Ana06!

**Review by mike-hunhoff — APPROVED** [review]
love it, nice work @Ana06!

**Pull request merged**
